test/rubygems/test_gem_commands_uninstall_command.rb

```diff
@@ -192,6 +192,62 @@ def test_execute_with_version_specified_as_colon
     assert File.exist? File.join(@gemhome, 'bin', 'executable')
   end
 
+  def test_uninstall_selection
+    ui = Gem::MockGemUi.new "1\ny\n"
```

Comment:
`y` is needless key type. I confirmed to pass the tests with `ui = Gem::MockGemUi.new "1\n"`